packages/valory/customs/balancer_pools_search/balancer_pools_search.py

```diff
@@ -422,12 +425,141 @@ def calculate_metrics(current_pool: Dict[str, Any], coingecko_api_key: str, coin
         il_risk_score = float('nan')
 
     sharpe_ratio = get_balancer_pool_sharpe_ratio(current_pool['pool_id'], current_pool['chain'].upper())  
-      
+    liquidity_risk_metric = analyze_pool_liquidity(current_pool['pool_id'])  
     return {
         "il_risk_score": il_risk_score,
-        "sharpe_ratio": sharpe_ratio
+        "sharpe_ratio": sharpe_ratio,
+        "liquidity_risk_metric":liquidity_risk_metric
```

Comment:
add two separate metrics, depth score and max_position_size instead of liquidity_risk_metric